artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/queue/StatQueue.java

```diff
@@ -67,7 +69,7 @@ String getJsonId() {
    private String value;
 
    @Option(name = "--maxRows", description = "max number of queues displayed. Default is 50.")
-   private int maxRows = 50;
+   private Integer maxRows;
```

Comment:
why change this?

Reply:
leave this as maxRows = DEFAULT_MAX_ROWS or 50./

Reply:
I would like to print the warning only if maxRows is not set by user.

Reply:
but surely the user anyhow would like to know if more rows than they gave anyhow right?

Reply:
I see your point but ATM the `listQueues(String options, int page, int pageSize)` return only maxRows queues. How can I help the users to know if there are more rows?

Reply:
I found a way using the attribute `count`. Let me try.

Reply:
sounds positive :)

Reply:
in your latest this can now just be reverted